keps/sig-cluster-lifecycle/20190227-artifact-generation.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Package Generation
+authors:
+  - "@klaven"
+  - "@ncdc"
+owning-sig: sig-cluster-lifecycle
+participating-sigs:
+  - sig-cluster-lifecycle
+  - sig-release
+reviewers:
+  - "@ncdc"
+  - "@hhorl"
+  - “@neolit123”
+  - “@ixdy”
+approvers:
+  - "@timothysc"
+editor: "@Klaven"
+creation-date: 2019-02-15
+last-updated: yyyy-mm-dd
+status: implementable
+see-also:
+  - "https://github.com/kubernetes/enhancements/pull/843"
+---
+
+# package-generation
+
+## Table of Contents
+
+- [Title](#title)
```

Comment:
I think the TOC is out of sync.

Reply:
will check it when I get a chance and update as needed.